common/output-model.cpp

```diff
@@ -994,22 +994,9 @@ void stream_dashboard::draw_dashboard(ux_window& win, rect& r)
     auto min_x = 0.f;
```

Comment:
min/max can be members to support separating this to functions that can be overloaded